graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -96,7 +93,7 @@ class SchemaElement(object):
     ELEMENT_KIND_EDGE = 'edge'
     ELEMENT_KIND_NON_GRAPH = 'non-graph'
 
-    def __init__(self, class_name, kind, abstract, properties, class_fields):
```

Comment:
Deleted class fields since they were not used neither in Defacto nor in the compiler.

Reply:
Since it relied on customFields from the OrientDB dict, deleting this made it easier to write the docs.